docs/examples/api-reference/sources/kafka.py

```diff
@@ -39,6 +39,62 @@ class SomeDataset:
     client.commit(message="some commit msg", datasets=[SomeDataset])
 
 
+@mock
+def test_kafka_source_with_secret(client):
+    # docsnip secret
+    from fennel.connectors import source, Kafka, Avro
+    from fennel.datasets import dataset, field
+    from fennel.integrations.aws import Secret
+
+    # docsnip-highlight start
```

Comment:
Nit: let's not highlight the whole section. Instead, let's highlight lines 50-53, 73-74 and 79-80.